examples/large_files/test/tests.ts

```diff
@@ -0,0 +1,126 @@
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
+import { Test, getCanisterId } from 'azle/test';
+import { Actor, ActorSubclass, HttpAgent } from '@dfinity/agent';
+import { hashFile } from 'azle/scripts/hash_file';
+
+export function getTests(canisterId: string): Test[] {
+    const origin = `http://${canisterId}.localhost:8000`;
+
+    return [
+        {
+            name: 'wait for things to finish uploading',
```

Comment:
Remove the need to wait here by polling in the post_install (really just in the npx azle upload-assets comand), every few seconds or so after all requests are sent, ask the canister if it's done, only stop processing when it's done